wvm-apps/wvm-exexed/crates/precompiles/src/inner/irys_precompile.rs

```diff
@@ -0,0 +1,37 @@
+use irys::irys::IrysRequest;
+use reth::primitives::Bytes;
+use reth::primitives::revm_primitives::{Precompile, PrecompileOutput, PrecompileResult};
+use reth::revm::precompile::{PrecompileWithAddress, u64_to_address};
+use reth_revm::primitives::B256;
+
+pub const PC_ADDRESS: u64 = 0x99;
```

Comment:
precompiles addresses should be incremental by 1 byte for each new address, so as we already have from 0x1 to 009 (1 to 9 bytes), this precompile should have a 10 bytes following the sequence which is 0x0A in hex instead of 0x99 (at address `0x000000000000000000000000000000000000000A`)

Reply:
P.S @andreespirela : instead of reserving the 10th precompile, and because Ethereum could add more precompiles in the future, and to maintain easy aligned with the sequencing, let's start our precompiles at index 23 instead of 9th. 23rd represent the position of letter "W" in the alphabet (WeaveVM) so that would be `0x17` (address: `0x0000000000000000000000000000000000000017`)